README.md

```diff
@@ -343,40 +374,39 @@ $ modality-probe-udp-collector
 Using the configuration:
     addr: 0.0.0.0:2718
     session id: 0
-    output file: /home/me/src/modality-probe/session_0_log_entries.jsonl
+    output file: /home/me/src/modality-probe/modality-probe-capi/session_0_log_entries.jsonl
 ```
 
 Then, in another terminal, navigate to the Rust example and run it.
 
 ```shell
-$ cd examples/rust-example
-$ cargo run
-    Finished dev [unoptimized + debuginfo] target(s) in 0.01s
-     Running `target/debug/rust-example`
-[2020-09-14T15:07:27Z INFO  rust_example] Modality probe reports will be sent to 127.0.0.1:2718
-[2020-09-14T15:07:27Z INFO  rust_example] Sensor measurement producer thread starting
-[2020-09-14T15:07:27Z INFO  rust_example] Sensor measurement consumer thread starting
-[2020-09-14T15:07:27Z INFO  rust_example] Consumer recvd 2
-[2020-09-14T15:07:27Z INFO  rust_example] All done
+$ cd ../examples/c-examples
+$ make run
+Modality probe reports will be sent to 127.0.0.1:2718
+Sensor measurement producer starting
+Sensor measurement consumer starting
+Consumer recvd -1
+Shutting down
+All done
 ```
 
-The `/home/me/src/modality-probe/session_0_log_entries.jsonl` file,
+The `/home/me/src/modality-probe/modality-probe-capi/session_0_log_entries.jsonl` file,
```

Comment:
Remove

Reply:
:+1: Reverting to:
The `/home/me/src/modality-probe/session_0_log_entries.jsonl` file,